React/main/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import authActions from '~/../redux/auth/actions';
+
+import style from './styles.scss';
+import LoginForm from './components/LoginForm';
+
+class Login extends Component {
+  attemptLogin = values => {
+    this.props.login(values);
+  };
+
+  render() {
+    const { loginError } = this.props;
+    return (
+      <div className={style['login-page']}>
+        <LoginForm onSubmit={this.attemptLogin} />
+        <div className={loginError ? style['login-error'] : style['login-error-hidden']}>
```

Comment:
You can write: { loginError && ... }. And it is not necessary have login-error-hidden class